File: py/IEC61970/Base/Core/ReportingGroup.py
# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106 on Fri Dec 15 17:03:56 2023
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Core.IdentifiedObject import IdentifiedObject
from CIM_STD_PYTHON.TC57CIM.IEC61970.Base.Core.PowerSystemResource import PowerSystemResource


class ReportingGroup(IdentifiedObject):
    """
    A reporting group is used for various ad-hoc groupings used for reporting.
    @author kdd
    @version 1.0
    @created 15-Dec-2023 4:38:29 PM
    """
    def __init__(self) -> None:
        super().__init__()
        self.power_system_resource = PowerSystemResource()

File: java/IEC61970/Base/Core/ReportingGroup.java
package TC57CIM.IEC61970.Base.Core;


/**
 * A reporting group is used for various ad-hoc groupings used for reporting.
 * @author kdd
 * @version 1.0
 * @created 15-Dec-2023 4:38:29 PM
 */
public class ReportingGroup extends IdentifiedObject {

	/**
	 * Power system resources which belong to this reporting group.
	 */
	public PowerSystemResource PowerSystemResource;

	public ReportingGroup(){

	}

	public void finalize() throws Throwable {
		super.finalize();
	}

}